src/api/migrations/20200622165311-create-commentReaction-table.js

```diff
@@ -0,0 +1,38 @@
+export default {
+    up: (queryInterface, Sequelize) => queryInterface.createTable('CommentReaction', {
+      id: {
+        allowNull: false,
+        autoIncrement: true,
+        primaryKey: true,
+        type: Sequelize.INTEGER
+      },
+      commentId: {
+        allowNull: false,
+        type: Sequelize.INTEGER,
+        references: { model: 'Comments', key: 'id' }
+      },
+      userId: {
+        allowNull: false,
+        type: Sequelize.INTEGER,
+        references: { model: 'Users', key: 'id' }
+      },
+      likes: {
```

Comment:
@MCFrank16 I just need to understand, why do we need likes and dislikes here?

Reply:
yeah I am using the same model as of @murediane for liking and disliking the comments, so I had to add in the column called ```reported``` to check if the comment is reported

Reply:
@MCFrank16 I'd suggest you follow the pattern of article reports. `Reported` shouldn't be in a commentReaction table that holds details of likes and dislikes for comments.